### PR TITLE
RDM-8494 CCD Definition Release 1.5.17

### PR DESCRIPTION
**Pull request by ckalista**
Release notes:

* ROC-7859: Changing the dashboard for counterclaim on paper which is received not via bulk scan - due in April 2020
* ROC-7427: CTSC Readiness - CCD event to allow defendant more time to respond.
* ROC-7950: ROC-7179  (change case state to 'settlementAgreementMade') incorrect in UAT Demo
* ROC-7698: Change name and description of judgeDecideAmount state
* ROC-7700: Decommission staff email: CCD changes for re-determination email.

### CODE REVIEW

File: definition/data/sheets/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "RejectOrganisationPaymentPlan", "UserRole": "citizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementRejectedByDefendant", "UserRole": "citizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementCounterSignedByDefendant", "UserRole": "citizen", "CRUD": "CR"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementSignedByBoth", "UserRole": "citizen", "CRUD": "CR"},
```
Comment (dharmendrak) [inline definition/data/sheets/AuthorisationCaseEvent.json:57]: this should be CR

Comment (daveoh) [reply definition/data/sheets/AuthorisationCaseEvent.json:57]: For production? Or only when CTSC goes live?

File: definition/data/sheets/AuthorisationCaseEvent.json [hunk 2]
```diff
@@ -212,7 +212,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "RejectOrganisationPaymentPlan", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementRejectedByDefendant", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementCounterSignedByDefendant", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "CR"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "AgreementSignedByBoth", "UserRole": "caseworker-cmc-anonymouscitizen", "CRUD": "CR"},
```
Comment (dharmendrak) [inline definition/data/sheets/AuthorisationCaseEvent.json:215]: this should also be CR if there is an existing process for it in SupportController

File: definition/data/sheets/AuthorisationCaseEvent.json (hunk 1 above)
Comment (ckalista) [reply definition/data/sheets/AuthorisationCaseEvent.json:57]: This is what is going to production next Monday. For citizen role it is generally safe to leave it as {{CR}}. Agreed?

Comment (daveoh) [reply definition/data/sheets/AuthorisationCaseEvent.json:57]: Agreed

File: definition/data/sheets/AuthorisationCaseEvent.json (hunk 2 above)
Comment (ckalista) [reply definition/data/sheets/AuthorisationCaseEvent.json:215]: Agreed.

Comment (daveoh) [reply definition/data/sheets/AuthorisationCaseEvent.json:215]: Yeah it looks like the auth should be `CR` for all roles

File: definition/data/sheets/AuthorisationCaseEvent.json (hunk 1 above)
Comment (ckalista) [reply definition/data/sheets/AuthorisationCaseEvent.json]: Code review addressed.

File: definition/data/sheets/AuthorisationCaseEvent.json (hunk 2 above)
Comment (ckalista) [reply definition/data/sheets/AuthorisationCaseEvent.json]: Code review comment addressed.

**Pull request merged**